jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -43,17 +44,31 @@
 - project:
     name: kubernetes-builds
     build:
-        - 'build':
-            branch: 'master'
-            timeout: 50
-        - 'build-1.0':
-            branch: 'release-1.0'
-            timeout: 30
-        - 'build-1.1':
-            branch: 'release-1.1'
-            timeout: 30
-        - 'build-1.2':
-            branch: 'release-1.2'
-            timeout: 30
+      - 'build':
+          url: 'https://github.com/kubernetes/kubernetes'
+          branch: 'master'
+          timeout: 50
+      - 'build-1.0':
+          url: 'https://github.com/kubernetes/kubernetes'
+          branch: 'release-1.0'
+          timeout: 30
+      - 'build-1.1':
+          url: 'https://github.com/kubernetes/kubernetes'
+          branch: 'release-1.1'
+          timeout: 30
+      - 'build-1.2':
+          url: 'https://github.com/kubernetes/kubernetes'
+          branch: 'release-1.2'
+          timeout: 30
+
+      - 'federation-build':
+          branch: 'federation-e2e'
+          url: 'https://github.com/colhom/kubernetes'
```

Comment:
Since we're not going to merge this until the kubernetes side is in, do we even need the url parameter?